### PR TITLE
use consistent style

### PR DESCRIPTION
**Pull request by wbclark**

### CODE REVIEW

File: spec/classes/plugin_certguard_spec.rb [hunk 1]
```diff
@@ -6,11 +6,12 @@
       let(:facts) { os_facts }
 
       it { is_expected.to compile.with_all_deps }
-      case os
-      when /\-7-/
-         it { is_expected.to contain_package('python3-subscription-manager-rhsm').with(ensure: 'present') }
+      it { is_expected.to contain_pulpcore__plugin('certguard') }
+      if facts[:os]['family'] == 'RedHat' && facts[:os]['release']['major'] == '7'
```
Comment (ekohl) [inline spec/classes/plugin_certguard_spec.rb:10]: The contents of `let` blocks are only available inside `it` blocks so you need to use the `os_facts` variable.
```suggestion
      if os_facts[:os]['family'] == 'RedHat' && os_facts[:os]['release']['major'] == '7'
```

**Comment by evgeni** [comment]
We can, but @jlsherrill wanted to do it in the installer -- you'd have to ask him why.

Also the package is needed on el8 too, just that there it's built by centos/rhel, not us.

**Comment by wbclark** [comment]
> We can, but @jlsherrill wanted to do it in the installer -- you'd have to ask him why.
> 
> Also the package is needed on el8 too, just that there it's built by centos/rhel, not us.

The reason for this is that the certguard plugin itself does not require `python3-subscription-manager-rhsm`, only the particular way Katello uses certguard requires that package.

**Comment by ekohl** [comment]
Could you elaborate? Is there a use case where the certguard plugin is useful without `python3-subscription-manager-rhsm` being installed?

**Comment by wbclark** [comment]
> Could you elaborate? Is there a use case where the certguard plugin is useful without `python3-subscription-manager-rhsm` being installed?

This is my understanding, though I'm not familiar with the details.

**Comment by evgeni** [comment]
I "think* we need it on el8 too, not 💯 sure

**Comment by jlsherrill** [comment]
yes, the pulp certguard plugin has two different 'guards', only one of them actually requires the rhsm package.  Thus the python package doesn't actually require it, and it seemed sensible to follow that through and not rquire it in the packaging.  

It is needed by el7 and el8, but for el7 we have to build it ourselves as @evgeni said

**Comment by jlsherrill** [comment]
@ekohl makes sense to me!

**Comment by ekohl** [comment]
https://github.com/theforeman/puppet-pulpcore/pull/103 fixes that. Let's merge that and rebase this.

**Comment by ekohl** [comment]
Mind rebasing this now?

**Pull request merged**